models/classes/media/sourceStrategy/HttpSource.php

```diff
@@ -114,12 +115,18 @@ public function getBaseName($link)
     }
 
     /**
-     * (non-PHPdoc)
+     * (non-PHPDoc)
      * @see \oat\tao\model\media\MediaBrowser::getDirectory()
      */
-    public function getDirectory($parentLink = '/', $acceptableMime = [], $depth = 1)
+    public function getDirectory(
+        string $parentLink = '',
+        array $acceptableMime = [],
+        int $depth = 1,
+        int $limit = 0,
+        int $offset = 0
+    ): array
     {
```

Comment:
[PSR-12](https://www.php-fig.org/psr/psr-2/#44-method-arguments): `When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.`
```suggestion
    ): array {
```